docs/website/package.json

```diff
@@ -15,7 +15,9 @@
     "debug": "DEBUG=* eleventy --input=src",
     "test": "echo \"Error: no test specified\" && exit 1",
     "prettier:write": "prettier --write 'src/**/*'",
-    "lint": "eslint -c .eslintrc.cjs src"
+    "lint:eslint": "eslint -c .eslintrc.cjs src",
```

Comment:
In a future PR, it probably makes sense to start using `wireit` like the rest of the repo.